src/components/Dashboard/Analytics/AnalyticsUtils.js

```diff
@@ -0,0 +1,121 @@
+// Barcharts
+
+export const barChartOptions = {
+    plugins: {
+        legend: {
+            position: "top",
+            display: false,
+        },
+        title: {
+            display: false,
+            text: "Activity",
+        },
+    },
+    maintainAspectRatio: false,
+    responsive: true,
+};
+
+export const mainBarChartDatasets = [
+    {
+        label: "Contributions days",
+        data: [345, 567, 765, 453, 676, 876, 564, 234, 440, 280, 440, 280],
```

Comment:
you should make it dynamic. if you want create another data file for this values and here use them in dynamic way.